writeengine/splitter/we_sdhandler.cpp

```diff
@@ -501,14 +501,9 @@ void WESDHandler::setup()
     //fLog.setLogFileName(aLogName.c_str(), aErrLogName.c_str(), false);
     fLog.setLogFileName(aLogName.c_str(), aErrLogName.c_str(), getConsoleLog());
 
-    // In mode 0 and Mode 1, we need to check for local file availability
-    if ((0 == fRef.fCmdArgs.getMode()) || (1 == fRef.fCmdArgs.getMode()))
-    {
-        if (!check4InputFile(fRef.getLocFile()))
-        {
-            throw (runtime_error("Could not open Input file " + fRef.getLocFile()));
-        }
-    }
```

Comment:
This used to do the existence/access check (wrongly).  Fixed it and moved it into setInputFileList().